### PR TITLE
chore: fix typos

### PR DESCRIPTION
**Pull request by xiaoxianBoy**
### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Review by imSanko — APPROVED** [review]
Those are some serious Typos

File: neural_network/convolution_neural_network.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def convolute(self, data, convs, w_convs, thre_convs, conv_step):
             )
             data_featuremap.append(featuremap)
 
-        # expanding the data slice to One dimenssion
+        # expanding the data slice to One dimension
```
Comment (tianyizheng02) [inline neural_network/convolution_neural_network.py:138]:
```suggestion
        # expanding the data slice to one dimension
```
Not strictly speaking a typo, but it's weird to have a single word capitalized for no good reason

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]
I found more typos but GitHub won't let me add comments on them:
- On line 79 of `points_are_collinear_3d.py`, "or" is misspelled as "of"
- On line 81 of that same file, "Since" is misspelled as "Sine"
- On line 4 of `convolution_neural_network.py`, "Handing Writing" should be "Handwriting"
- On line 307 of that same file, it says "Training Complished" but "Complished" isn't a real word; I think it's supposed to say "Complete"?
- On line 356 of that same file, it says "on other file" but I think it's supposed to say "in another file"? I'm not sure what file the author was referring to

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Were these misspellings found with a tool?

**Comment by tianyizheng02** [comment]
> Were these misspellings found with a tool?

No, I just visually inspected the files.

**Pull request merged**